projects/ccf-eui/src/themes/_default_light.scss

```diff
@@ -1,14 +1,13 @@
 @use '@angular/material' as mat;
-@use '@angular/material/theming' as *;
 @use 'common/utility';
 @import 'light';
 
 $default-light-palette: utility.add-colors($primary-colors, (
   main: #444A65,
   light-disabled: #E0E0E0,
   contrast: (
-    main: $dark-primary-text,
-    light-disabled: $dark-primary-text,
+  //   main: $dark-primary-text,
```

Comment:
What happened here?

Reply:
Angular upgrade broke it so I disabled it